src/components/common/DataTable.vue

```diff
@@ -99,10 +99,10 @@ export default class DataTable extends Vue {
   }
 
   get paginatedItems() {
-    return this.modifiedItems.slice(
-      this.currentPage * this.perPage - this.perPage,
-      this.currentPage * this.perPage
-    );
+    const perPage = this.perPage;
+    const endIndex = this.currentPage * perPage;
+    const startIndex = endIndex - perPage;
+    return this.modifiedItems.slice(startIndex, endIndex);
```

Comment:
In getters it's best to only reference outside data like component variables or other getters just the once. If the getter were to run often enough it will be better for performance.

Reply:
cool, thanks for the feedback. Will make sure to always do this going forward.